app/views/movie/create.scala.html

```diff
@@ -0,0 +1,42 @@
+@import helper._
+@import modules.utility.twirl.CustomHelpers._
+@import modules.movie.{CreateMovieForm}
+@import modules.util.{Country, Language}
+@(movieForm: Form[CreateMovieForm], id: Option[Long] = None)(implicit request: MessagesRequestHeader, flash: Flash)
+
+@main("Create movie") {
+    @form(
+        if (id != None) modules.movie.routes.MovieController.update(id.get) else modules.movie.routes.MovieController.saveNewMovie(),
```

Comment:
Although this will never produce runtime errors, it's more expressive to not use `if (Option.nonEmpty) Option.get`, and instead use `Option.map(doSomething).getOrElse(doSomethingElse)`.